builder/vmware/iso/driver_esx5.go

```diff
@@ -221,6 +221,8 @@ func (d *ESX5Driver) VNCAddress(_ string, portMin, portMax uint) (string, uint,
 			if e, ok := err.(*net.OpError); ok {
 				if e.Timeout() {
 					log.Printf("Timeout connecting to: %s (check firewall rules)", address)
+				} else if strings.Contains(e.Error(), "connection refused") {
```

Comment:
this looks like a different code path from what I see in the logs. Does this line impact you specifically?